src/tests/topp/FeatureFinderMetabo_2_noEPD_output.featureXML

```diff
@@ -5,14 +5,14 @@
 		<processingAction name="Quantitation" />
 		<userParam type="string" name="parameter: mode" value="test_mode"/>
 	</dataProcessing>
-	<featureList count="16">
+	<featureList count="13">

```

Comment:
seems like a big change of default parameters here? 
why not also have a TOPP test for the old and the new parameter set?